graders/r/plr/ChangeLog

```diff
@@ -0,0 +1,50 @@
+2021-08-31  Dirk Eddelbuettel  <edd@debian.org>
```

Comment:
We could continue to maintain a ChangeLog, or decide we're done with it (and live by git commit messages) and delete this one.

Reply:
Updated.